pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx

```diff
@@ -24,27 +29,68 @@ export const AdditionalOptionsFormPart: FC<AdditionalOptionsFormPartProps> = ({
         />
         <CheckboxField label={Messages.form.labels.additionalOptions.tls} name="tls" />
         <CheckboxField label={Messages.form.labels.additionalOptions.tlsSkipVerify} name="tls_skip_verify" />
-        {getAdditionalOptions(instanceType, remoteInstanceCredentials)}
+        {getAdditionalOptions(instanceType, remoteInstanceCredentials, form)}
       </div>
     </div>
   );
 };
+
 export const PostgreSQLAdditionalOptions: FC<PostgreSQLAdditionalOptionsProps> = () => (
-  <Field
-    dataQa="tracking-options-radio-button-group"
-    name="tracking"
-    label={Messages.form.labels.trackingOptions}
-    options={trackingOptions}
-    component={RadioButtonGroupAdapter}
-  />
+  <>
+    <Field
+      dataQa="tracking-options-radio-button-group"
+      name="tracking"
+      label={Messages.form.labels.trackingOptions}
+      options={trackingOptions}
+      component={RadioButtonGroupAdapter}
+    />
+  </>
 );
-export const getAdditionalOptions = (type, remoteInstanceCredentials) => {
+
+const MySQLOptions = ({ form }) => {
+  const [selectedValue, setSelectedValue] = useState<string>(TablestatOptions.disabled);
+
+  const getTablestatValues = (type) => {
```

Comment:
Function can be moved to outside of the component.

Reply:
done